client/src/templates/Challenges/quiz/show.tsx

```diff
@@ -0,0 +1,278 @@
+    } = this.props;
+    if (prevTitle !== currentTitle) {
+      updateChallengeMeta({
+        ...challengeMeta,
+        title: currentTitle,
+        challengeType,
+        helpCategory
+      });
+      challengeMounted(challengeMeta.id);
+    }
+  }
+
+  handleSubmit() {
+    console.log('handleSubmit');
+  }
+
+  render() {
+    const {
+      data: {
+        challengeNode: {
+          challenge: {
+            title,
+            // challengeType,
+            description,
+            superBlock,
+            block,
+            translationPending,
+            fields: { blockName },
+            quizzes
+          }
+        }
+      },
+      // openCompletionModal,
+      openHelpModal,
+      pageContext: {
+        challengeMeta: { nextChallengePath, prevChallengePath }
+      },
+      t,
+      isChallengeCompleted
+    } = this.props;
+
+    const blockNameTitle = `${t(
+      `intro:${superBlock}.blocks.${block}.title`
+    )} - ${title}`;
+
+    const random = Math.floor(Math.random() * quizzes.length);
+    const quiz = quizzes[random].questions;
+    const quizForComponent = quiz.map(question => {
+      return {
+        question: question.question,
+        answers: question.options.map((option, index) => {
+          return {
+            label: option,
+            value: index + 1
+          };
+        })
+      };
+    });
+
+    return (
+      <Hotkeys
+        executeChallenge={() => {
+          this.handleSubmit();
+        }}
+        containerRef={this.container}
+        nextChallengePath={nextChallengePath}
+        prevChallengePath={prevChallengePath}
+      >
+        <LearnLayout>
+          <Helmet
+            title={`${blockNameTitle} | ${t('learn.learn')} | freeCodeCamp.org`}
+          />
+          <Container>
+            <Row>
+              <Spacer size='medium' />
+              <ChallengeTitle
+                isCompleted={isChallengeCompleted}
+                translationPending={translationPending}
+              >
+                {title}
+              </ChallengeTitle>
+
+              <Col md={8} mdOffset={2} sm={10} smOffset={1} xs={12}>
+                <ChallengeDescription description={description} />
+                <ObserveKeys>
+                  <Quiz questions={quizForComponent} />
+                </ObserveKeys>
+                <Spacer size='medium' />
+                <Button
+                  block={true}
+                  variant='primary'
+                  onClick={() => this.handleSubmit()}
+                >
+                  {t('buttons.check-answer')}
+                </Button>
+                <Spacer size='xxSmall' />
+                <Button block={true} variant='primary' onClick={openHelpModal}>
+                  {t('buttons.ask-for-help')}
+                </Button>
+                <Spacer size='large' />
+              </Col>
+              <CompletionModal />
+              <HelpModal challengeTitle={title} challengeBlock={blockName} />
```

Comment:
Do we want to show the help modal here?

I am leaning towards leaving it out since this is a quiz.